src/contract_clients/utils.rs

```diff
@@ -26,23 +30,26 @@ pub async fn build_single_owner_account<'a>(
     account_address: &str,
     is_legacy: bool,
 ) -> RpcAccount<'a> {
-    let signer = LocalWallet::from(SigningKey::from_secret_scalar(FieldElement::from_hex_be(private_key).unwrap()));
-    let account_address = FieldElement::from_hex_be(account_address).expect("Invalid Contract Address");
-    let execution_encoding = if is_legacy {
-        starknet_accounts::ExecutionEncoding::Legacy
-    } else {
-        starknet_accounts::ExecutionEncoding::New
-    };
+    let signer = LocalWallet::from(SigningKey::from_secret_scalar(Felt::from_hex(private_key).unwrap()));
```

Comment:
(nit) Cloning would help use removing this lifetime everywhere, right? Should probably be done at some point, the bootstraper doesn't really care about this small perf gain and the code would be simpler (in other places too)

Reply:
yes that's true